config.go

```diff
@@ -32,6 +37,8 @@ type Config struct {
 	// Consul. This requires Consul to be configured to serve HTTPS.
 	SSL bool `mapstructure:"ssl"`
 
+	HttpAuth HttpAuth `mapstructure:"HttpAuth"`
```

Comment:
But if we follow the "single-value" for authentication, this would likely just become a string in the config.